src/PlaywrightSharp/Playwright.cs

```diff
@@ -97,14 +97,23 @@ internal Playwright(IChannelOwner parent, string guid, PlaywrightInitializer ini
         /// <param name="browsersPath">Specify a shared folder that playwright will use to download browsers and to look for browsers when launching browser instances.
         /// It is a shortcut to the PLAYWRIGHT_BROWSERS_PATH environment variable.
         /// </param>
+        /// <param name="debug">Enabled the playwright driver log.
+        /// It is a shortcut to the DEBUG=pw:api environment variable.
+        /// </param>
         /// <returns>A <see cref="Task"/> that completes when the playwright driver is ready to be used.</returns>
         public static async Task<IPlaywright> CreateAsync(
             ILoggerFactory loggerFactory = null,
             TransportTaskScheduler scheduler = null,
             string driversLocationPath = null,
             string driverExecutablePath = null,
-            string browsersPath = null)
+            string browsersPath = null,
+            bool debug = false)
         {
+            if (debug)
+            {
+                Environment.SetEnvironmentVariable("DEBUG", "pw:api");
```

Comment:
Consider having the debug flag be a string to be used as the DEBUG env var

Reply:
I like it!